### DISCUSSION TITLE
I accidentally entered my wallet address as a private key and new wallet address was created.

### DISCUSSION THREAD

**Discussion by tunavet06** [aptos-labs/aptos-developer-discussions#531] [Questions]
### Discord user ID

@tunavet

### Describe your question in detail.

Hello, while I was trying to open a new wallet and make a transaction, I accidentally entered my wallet address as a private key and new wallet address was created. I made 35.5 apt transfers to this wallet via my address and now I cannot get them back, can you please help.

### What error, if any, are you getting?

cant send my tokens

### What have you tried or looked at?  Or how can we reproduce the error?

send back my aptos coins

### Which operating system are you using?

Windows

### Which SDK or tool are you using? (if any)

TypeScript SDK

### Describe your environment or tooling in detail

Hello, while I was trying to open a new wallet and make a transaction, I accidentally entered my wallet address as a private key and new wallet address was created. I made 35.5 apt transfers to this wallet via my address and now I cannot get them back, can you please help.

**Comment by 0xbrw** [comment]
have you tried restoring your wallet with your recovery phrase?

**Comment by Pcharlesme** [comment]
let me explain Private Key vs. Wallet Address: A wallet address is derived from a private key. However, using a wallet address as a private key is invalid, and the resulting wallet address is unpredictable. This means you don't actually "own" the new wallet because there’s no way to derive the correct private key for it.

try 
Recover the Wallet 
Use an Aptos blockchain explorer to verify the balance and transaction history of the wallet address
